account_analytic_no_lines/tests/test_account_analytic_no_lines.py

```diff
@@ -0,0 +1,117 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openerp.tests.common import SavepointCase
+
+
+class TestAccountAnalyticNoLines(SavepointCase):
+
+    @classmethod
+    def setUpClass(cls):
```

Comment:
Therefore what's the problem? At the moment you are right we don't take advantage of the use of ` SavepointCase` because there's only one test. Nevertheless it is not a bad practice and it can not block the PR. Moreover we should add this method in the OCA guidelines as good practice.